content/pages/education/gi-bill-school-feedback.md

```diff
@@ -0,0 +1,7 @@
+---
+title: GI Bill® School Feedback Tool
+entryname: complaint-tool
+layout: page-react.html
+description: Use your VA education benefits to pay for college or training programs. Find out which documents you’ll need to apply for benefits, and start your online application today.
+includeBreadcrumbs: true
```

Comment:
It looks like the nav is removed. Is this a new pattern, or should it be included?

Reply:
I believe that adding this `includeBreadcrumbs` is the new normal. I forget where I saw that. It's a recent change. But the nav does indeed show up.

Reply:
I also made the corresponding change to the Yeoman generator so that it does this for the new forms it creates.

Reply:
Thanks for the context.